src/Sylius/Behat/Context/Ui/CartContext.php

```diff
@@ -46,6 +46,7 @@ public function __construct(
 
     /**
      * @Given I added product :product to the cart
+     * @Given I had product :product in the cart
      * @When I add product :product to the cart
      * @When I have product :product in the cart
```

Comment:
Do we have a use case when `When I have product ...` makes sense to be used instead of `When I add product ...`?